crates/aiken-lang/src/format.rs

```diff
@@ -1110,6 +1114,27 @@ impl<'comments> Formatter<'comments> {
         }
     }
 
+    fn logical_op_chain<'a>(
+        &mut self,
+        kind: &'a LogicalOpChainKind,
+        expressions: &'a [UntypedExpr],
+    ) -> Document<'a> {
+        kind.to_doc()
+            .append(" {")
```

Comment:
This should probably be optional if the child expression is already a block?

Reply:
Interesting, I don't understand but I want to. Can you explain more?